.circleci/config.yml

```diff
@@ -0,0 +1,82 @@
+---
+version: 2.1
+
+orbs:
+    slack: circleci/slack@3.4.2
+
+executors:
+  builder:
+    docker:
+      - image: circleci/php:latest
+  publisher:
+    docker:
+      - image: gcr.io/google.com/cloudsdktool/cloud-sdk:alpine
+        auth:
+          username: _json_key
+          password: $GCLOUD_GCR_ADMIN_KEY
+
+jobs:
+  build:
+    executor: builder
+    working_directory: ~/prestashop
+    steps:
+      - checkout
+      - run:
+          name: Build Package
+          command: sh packlink-build.sh ${CIRCLE_TAG}
+      - persist_to_workspace:
+          root: .
+          paths:
+            - artifact.zip
+  publish:
+    executor: publisher
+    working_directory: ~/prestashop
+    steps:
+      - attach_workspace:
+          at: .
+      - run:
+          name: Configure gcloud
+          command: |
+            echo ${PRESTASHOP_GCLOUD_SERVICE_KEY} | base64 -d | gcloud auth activate-service-account --key-file=-
+            gcloud --quiet config set project ${PRESTASHOP_GOOGLE_PROJECT_ID}
+            gcloud --quiet config set compute/zone ${PRESTASHOP_GOOGLE_COMPUTE_ZONE}
```

Comment:
posh comment about style: same as above, CircleCI avoids to use PRESTASHOP in its doc